gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -82,9 +82,11 @@ class VeloxTestSettings extends BackendTestSettings {
     // NEW SUITE: disable as it expects exception which doesn't happen when offloaded to gluten
     .exclude(
       "INCONSISTENT_BEHAVIOR_CROSS_VERSION: compatibility with Spark 2.4/3.2 in reading/writing dates")
-    .exclude("FAILED_EXECUTE_UDF: execute user defined function")
+    // JDBC test
```

Comment:
Can you provide a detailed explanation as to why this suite needs to be excluded?

Reply:
This test needs to be excluded because it is jdbc related test which uses jdbc driver class which is not needed in gluten.This was also a new test in 3.4